tests/integration_tests/test_mongo_task.py

```diff
@@ -261,6 +264,7 @@ def test_composite_key_erasure(
     assert access_request_data["mongo_test:composite_pk_test"][0]["description"] == None
 
 
+@pytest.mark.integration_mongodb
```

Comment:
class `TestRetrievingDatMongo` below could be marked at the class level, and additional marked with mongo -